zipline/pipeline/loaders/quarter_estimates.py

```diff
@@ -0,0 +1,582 @@
+from collections import defaultdict
```

Comment:
Is it really only for earnings estimates, though? Can't this be for estimates of other values associated with a quarterly event?